freshmaker/image.py

```diff
@@ -361,8 +361,13 @@ def resolve_content_sets(self, pyxis_api_instance, children=None):
     def resolve_published(self, pyxis_api_instance):
         # Get the published version of this image to find out if the image
         # was actually published.
-        images = pyxis_api_instance.get_images_by_nvrs([self.nvr], published=True)
-        if images:
+        images = pyxis_api_instance.find_images_by_nvr(self.nvr)
+        if not images:
+            log.warning("No image %s found in Pyxis.", self.nvr)
+            return
+
+        published = any([r["published"] is True for img in images for r in img["repositories"]])
```

Comment:
Could we omit `is True` ?
```suggestion
        published = any([r["published"] for img in images for r in img["repositories"]])
```

Reply:
You're right, updated.